module/applications/advancement/scale-value-config.mjs

```diff
@@ -47,8 +71,11 @@ export default class ScaleValueConfig extends AdvancementConfig {
     let levels = Array.fromRange(CONFIG.DND5E.maxLevel + 1);
     if ( ["class", "subclass"].includes(this.advancement.item.type) ) levels = levels.slice(1);
     return levels.reduce((obj, level) => {
-      obj[level] = { placeholder: this._formatPlaceholder(lastValue), value: null };
-      const value = this.advancement.configuration.scale[level];
+      const value = this.advancement.configuration.scale[level]?.clone();
```

Comment:
We didn't have to clone before?

Reply:
I believe we previously had a small issue where `_mergeScaleValues` was making changes to the data on the advancement rather than just preparing changes for preview.